packages/dashboard/src/App.tsx

```diff
@@ -16,6 +26,12 @@ i18n.use(initReactI18next)
 addMaskThemeI18N(i18n)
 addDashboardI18N(i18n)
 applyMaskColorVars(document.body, 'light')
+// TODO:
```

Comment:
nit: Please comment what to do. It would be helpful for code-reviewing

Reply:
Oh, it's obvious, as you can see the `eth` and `enabled` implementation are empty implementations.